README.md

```diff
@@ -1 +1,56 @@
-# Hello-world
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] Pull Request should have a summary describing introduced changes in more detail, please add more info about the changes that you made in this PR

Reply:
thanks, I got it!